gitbook-docs/market/overview.md

```diff
@@ -1,63 +1,84 @@
+all additional metadata to it.
+See the [prepare statement](cmd-reference/statement.md/#prepare-statement) section.
 
 {% hint style="info" %}
-The instruments will require approval by =nil;Foundation to trade on Proof Market.
+To be traded on the Proof Market, the instruments will require approval by `=nil;` Foundation.
 {% endhint %}
 
-
-
 ## Entities
 
-The marketplace consists of the following entities.
-
-* **Proof Requester**: This can be an application like a bridge requesting balance or a user interested in the cross-cluster operation and/or trust-less data access.
-* **Proof Producer**: This is an entity that will generate the proofs for the requests/orders made by the Proof Requester.
-
-## Orders Types
-
-* **Buy Order**: A buy order is posted by a proof requester. Each buy order comprises of following parameters:
-  * **Circuit**: The instruments for which this order is placed. ex: MINA state-proof circuit or Solana state-proof circuit.
-  * **Cost**: Amount the user is willing to pay for the proof. This is a number represented in USD.
-  * **Order Time Out**: Once the order enters the marketplace, this defines how long it should stay in the order book. If it times out before being matched, it is purged. This is a number expressed in minutes.
-  *   **Generation Time**:  Once the order is matched, this specifies the maximum time the proof generator has to publish the proof. This is an optional parameter. This is a number expressed in minutes.&#x20;
-
-
-* **Sell Order**: A sell order is posted by a proof Generator. Each sell order comprises of following parameters.
-  * **Circuit**: The instruments for which a generator is willing to generate a proof.
-  * **Cost:** Amount the proof generator is willing to accept to create a proof.
-  * **Order Time Out**: Once the order enters the marketplace, this defines how long it should stay in the order book. If it times out before being matched, it is purged. This is a number expressed in minutes.
-  * **Generation Time:** The upper bound by which the proof will be generated once matched.
-
-## Order Matching
-
-Currently, a daemon process matches the orders when the following parameters are met within the order timeout criteria.
-
-* Cost
-* Generation time
-
-## Order Settlement
-
-Order settlement takes place once the proof generator has submitted the proof to the marketplace after generation.
-
+The marketplace consists of the following entities:
+
+* **Proof requester** — an application like a bridge requesting balance or a user
+    interested in a cross-cluster operation or trustless data access.
+* **Proof producer** — an entity that will generate the proofs for the requests/orders
+    made by the proof requester.
+
+## Orders types
+
+* **Buy order**: A buy order is posted by a proof requester.
+    Each buy order comprises the following parameters:
+  * **Statement** (JSON) — the instrument for which this order is placed, like MINA or Solana
```

Comment:
Order only points to the statement and contains its id, not the whole description

Reply:
@SK0M0R0H statement id then? Like this:
**Statement id** — the id of the instrument for which this order is placed, like MINA or Solana state-proof circuit.

Reply:
Sorry, "id" was more like the idea. Technically, it's called key now in user scripts